src/locales.ts

```diff
@@ -0,0 +1,52 @@
+// WARNING: This code is generated by build-locales.js. Please do not modify it directly (including formatting).
+export const locales: string[] = [
+    'af',
+    'ar',
+    'az',
+    'be',
+    'bg',
+    'ca',
```

Comment:
Thanks to autogeneration, the list will be automatically maintained up-to-date.

For example, the Catalan locale (`ca`) is supported in this package, but is not added to the locales list in the theme-kit package.